entanglement_forging/core/wrappers/entanglement_forged_vqe_result.py

```diff
@@ -123,6 +123,9 @@ def __init__(
         energy_std_each_parameter_set: Optional[List[Union[float, int]]] = None,
         energy_offset: Optional[float] = None,
         eval_count: Optional[int] = None,
+        num_particles: Optional[Tuple[complex, complex]] = None,
```

Comment:
I would expect this to be a real positive integer. Could potentially update type to reflect this.

Reply:
Does it have to be an integer though? What if the state we are evaluating is some superposition of states with different number of particles? Unless we are 100% sure that this is not going to happen generally for any problem, we should leave it a real instead of an integer.

Reply:
Yes, I think you're right @pedrorrivero